src/core.js

```diff
@@ -95,8 +95,11 @@ export function updateCursor(event, originalValue, originalPosition) {
     return
   }
 
+  // if event.inputType is not supported, assume 'insertText'
+  const inputType = event.inputType || 'insertText'
+
   // get some information about the cursor based on the original value
-  const isInsertEvent = ['insertText', 'insertFromPaste'].includes(event.inputType)
+  const isInsertEvent = ['insertText', 'insertFromPaste'].includes(inputType)
   const wasCursorAtEnd = isInsertEvent && originalPosition == originalValue.length
   let lastInsertedChar = isInsertEvent && originalValue[originalPosition - 1]
```

Comment:
Seems like `wasCursorAtEnd` and `lastInsertedChar` could be wrong if the input was delete on IE. Wouldn't that mess things it up?

Reply:
No issues there. Backspacing works just fine in IE with these changes.

Reply:
[Apparently they didn't support IE.](https://github.com/RonaldJerez/vue-input-facade/issues/26#issuecomment-689251164) That may not bode well for them accepting your PR.

I feel certain there are some edge case bugs, otherwise what's the point of specifying `insertText`? After testing a bit in production it seems `wasCursorAtEnd` is important for deleting in the middle of the string. When I deleted a digit in the middle of the phone number the cursor jumped to the end. Also, afterwards I hit undo and several numbers were added to the end of the string. 

What do you think of mitigating some of the stranger behavior by suppressing some functionality in IE. For example, not allowing undo in IE for masked inputs. It would be slightly frustrating but it wouldn't seem haunted. I'm not sure what we could do about the cursor jumping to the end. Another idea is we could not use facade for IE and just make the user punch in all the characters. The main reason we're using a mask is to make input easy on a mobile device.

Reply:
The only edge case I've encountered after this change is the undo behavior you mentioned, which also exists prior to the change. Deleting in the middle of the text seems to be working correctly with these changes.